### ISSUE TITLE
react/jsx-indent conficts with react/jsx-closing-bracket-location

### ISSUE THREAD

**Issue by apexad** [yannickcr/eslint-plugin-react#1120]
As of v6.10.1 (no issue in v6.10.0) there is a conflict in these 2 rules:

`'react/jsx-indent-props': ['error', 2],`
`'react/jsx-closing-bracket-location': ['error', 'props-aligned'],`
`
```javascript
  <Router history={browserHistory}>
    <Route
      path="/"
      getComponent={getMain}
      >
```
`react/jsx-indent` wants 2 spaces deleted before the last >, but then this triggers `react/jsx-closing-bracket-location`

**Comment by apexad** [comment]
Actually appears fixed in 6.10.3
